lib/experimental/auto-inserting-blocks.php

```diff
@@ -0,0 +1,246 @@
+<?php
+/**
+ * Auto-inserting blocks.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Return a function that auto-inserts blocks relative to a given block.
+ *
+ * @param array  $inserted_block    The block to insert.
+ * @param string $relative_position The position relative to the given block.
```

Comment:
a quick example could remove a lot of guessing, or a list of valid values.

```
 * @param string $relative_position Where to insert the block with respect to the anchor block; one of 'first_child', 'last_child', 'before', 'after'.
```

Reply:
Yes! While I did add [something to that effect](https://github.com/WordPress/gutenberg/pull/51449/files#diff-b6a5f5374aa2b01e4369049937355b47072d9123e8918523bc49b46efa501807R125) to `gutenberg_register_auto_inserted_block`, I missed this occurrence. I'll do so now (in a follow-up PR).

Reply:
10bf692288